test/functional_cpu_test.py

```diff
@@ -300,19 +300,23 @@ def test_linearity_of_istft4(self):
 
 class TestDetectPitchFrequency(common_utils.TorchaudioTestCase):
     def test_pitch(self):
-        test_filepath_100 = common_utils.get_asset_path("100Hz_44100Hz_16bit_05sec.wav")
-        test_filepath_440 = common_utils.get_asset_path("440Hz_44100Hz_16bit_05sec.wav")
+        SAMPLE_RATE = 44100
```

Comment:
Why is this variable capitalized?

Reply:
I was thinking this is a constant per https://www.python.org/dev/peps/pep-0008/#id48?  What case would you prefer?